sdks/python/apache_beam/ml/transforms/embeddings/vertex_ai.py

```diff
@@ -73,6 +89,8 @@ def __init__(
     self.task_type = task_type
     self.title = title
 
+  @retry.with_exponential_backoff(
+      retry_filter=retry_on_vertex_ai_errors, num_retries=5)
```

Comment:
IIUC, this will retry on entire batch rather just the failed elements? We could potentially separated the VertexAI call and decorate that method instead like https://github.com/apache/beam/blob/89d1c06e1eab530c4e95a807a035b450c340e5f8/sdks/python/apache_beam/ml/inference/vertex_ai_inference.py#L193